src/chainparams.cpp

```diff
@@ -616,6 +619,7 @@ class CRegTestParams : public CChainParams {
 
         /** RingCT/Stealth **/
         nDefaultRingSize = 11;
+        nMinimumStakeQuantity = 1 * COIN;
```

Comment:
It's become common practice to have these constants defined separately in regtest and testnet and mainnet.  The main code calls a wrapper routine to pull whichever applies for the network being run, allowing for common code and runtime differences (between them) rather than a recompile that a constant would require .